### PR TITLE
[WIP] docs(adopters): add Gitpod to the adopters page

### PR DESCRIPTION
**Pull request by ghuntley**
## Description

Provides a way for companies to publicly share that they are using Gitpod. 

See [discussion](https://gitpod.slack.com/archives/C01LDA27WJC/p1636537578243100?thread_ts=1636533879.238500&cid=C01LDA27WJC) before merging.

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **geropl** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @geropl` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by loujaybee — DISMISSED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 66fae60c4262595f7d7ec28b5bf824941288ebfa</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6641#pullrequestreview-802339440" title="Approved">loujaybee</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [loujaybee]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: ADOPTERS.md
Comment (loujaybee) [inline ADOPTERS.md:4]: [The Backstage one](https://github.com/backstage/backstage/blob/master/ADOPTERS.md) also has a column for an optional contact? Should/could we add that also? That way if companies want to get in touch with others about how they're using Gitpod that they can do it? Just to further promote the cross-company communication?

Comment (loujaybee) [inline ADOPTERS.md:4]: [The Backstage one](https://github.com/backstage/backstage/blob/master/ADOPTERS.md) also has a column for an optional contact? Should/could we add that also? That way if companies want to get in touch with others about how they're using Gitpod that they can do it? To further promote the cross-company communication?

**Comment by loujaybee** [comment]
This is awesome, thanks for setting up @ghuntley !! Love to see Gitpod as the first on the list, too 😍

**Comment by roboquat** [comment]
New changes are detected. LGTM label has been removed.

File: ADOPTERS.md
Comment (ghuntley) [reply ADOPTERS.md:5]: 👍 added instructions of use.

File: ADOPTERS.md
Comment (MvRemmerden) [inline ADOPTERS.md:4]:
```suggestion
| [GitLab](https://gitlab.com)                                             | [@gitlab](https://www.twitter.com/gitlab)                                                                                                                                                                    | Developing and reviewing changes for GitLab, the handbook, marketing page and documentation.                                                                                                                                                                                                                                                        |
| <the name of your company and a link to the website>                     | <a name or handle of a person and link to a social media account>                                                                                                                                            | <describe how you are using Gitpod>                                                                                                                                                                                                                                                                                                                  |
``` 

@ghuntley Love the idea, how about already adding GitLab as first adopter here? 😉

**Review by MvRemmerden — APPROVED** [review]

**Comment by roboquat** [comment]
@MvRemmerden: changing LGTM is restricted to collaborators

<details>

In response to [this](https://github.com/gitpod-io/gitpod/pull/6641#pullrequestreview-802866650):

>


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6641#pullrequestreview-802339440" title="Approved">loujaybee</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/6641#pullrequestreview-802866650" title="Approved">MvRemmerden</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [loujaybee]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by 0Grit** [comment]
Do I need to sign the CLA first before declaring adoption?

**Comment by mikenikles** [comment]
Hi @ghuntley, what's the status with this PR? It currently blocks https://github.com/gitpod-io/website/pull/1235 which updates the docs on the website.

I'm going to move the docs back into the `gitpod-io/gitpod` repo and if the 1235 PR in the website repo can't get merged ahead of the docs migration, you'll have to manually copy the code to the new docs location.

**Comment by loujaybee** [comment]
Closing, related comment: 

https://github.com/gitpod-io/website/pull/1235#issuecomment-1169907993

**Pull request closed without merging**
